### ISSUE TITLE
[Accessibility]: Navigating to blob view requires counter-intuitive steps

### ISSUE THREAD

**Issue by varungandhi-src** [sourcegraph/sourcegraph#36531]
### Audit type

Screen reader navigation

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34448

### Problem description

The blob view seems to have a table inside a group, making it tricky to go to the first line quickly. You need to Tab to the group, then Navigate to Prev Line (Control + Option + ←) then Navigate to Next Line (Control + Option + →). Trying Navigate to Next Line directly from the group takes one to the next item (e.g. a git blame URL if the Git extension is enabled).

### Expected behavior

Once the focus is on the blob view as a whole, I think the right behavior would be that Navigate to Next Line would take you to the first line of the blob view.

I'm not sure if that means that we should remove the group or do something else. The group doesn't seem to be adding any functionality visually. 🤔

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by felixfbecker** [comment]
Closing as the CodeMirror blob no longer uses a `<table>`
